src/main/resources/io/viash/runners/nextflow/states/findStates.nf

```diff
@@ -82,8 +82,8 @@ def findStates(Map params, Map config) {
           // construct renameMap
           if (args.rename_keys) {
             def renameMap = args.rename_keys.collectEntries{renameString ->
-              def split = renameString.split(";")
-              assert split.size() == 2: "Argument 'rename_keys' should be of the form 'newKey:oldKey,newKey:oldKey'"
+              def split = renameString.split(":")
+assert split.size() == 2: "Argument 'rename_keys' should be of the form 'newKey:oldKey', or 'newKey:oldKey;newKey:oldKey' in case of multiple values"
```

Comment:
```suggestion
              assert split.size() == 2: "Argument 'rename_keys' should be of the form 'newKey:oldKey', or 'newKey:oldKey;newKey:oldKey' in case of multiple values"
```

Reply:
Sall good!